packages/bot/src/constants/index.js

```diff
@@ -271,6 +271,11 @@ const config = {
             description: localize('The Oscar\'s Grind Strategy is a low-risk positive progression strategy that first appeared in 1965. By using this strategy, you will increase the size of your contract after each successful trade, the size of your contract will increase after successful trades, and remain unchanged after unsuccessful trades.'),
         },
     },
+    save_status: {
+        EMPTY : { index: 0, message: localize('') },
+        SAVING: { index: 1, message: localize('Saving draft') },
```

Comment:
Perhaps we could add three dots indicating progress.

```suggestion
        SAVING: { index: 1, message: localize('Saving draft...') },
```